src/commands/audit.js

```diff
@@ -31,5 +31,5 @@ const parserVersion = require('../parser-version');
  */
 module.exports = function(opts) {
   parserVersion();
-  mvnwSync(['--version']);
+  mvnw(['--version']);
```

Comment:
@volodya-lombrozo why don't you use `then` here? Without it, if I understand correctly, the execution in this parallel "process" will still be running, while the function the `audit` will exist.

Reply:
@yegor256 We can use [`Promises` without then](https://stackoverflow.com/questions/36211456/promises-without-then) and everything will works fine. Also here is no "parallelism" at all (only async invocations). Moreover, our tests prove that everything works fine. Anyway I've added the `return` statement for `audit` to give a caller a chance to call `.then` - perhaps it will be useful in the future.